src/coffee/run.coffee

```diff
@@ -308,6 +309,7 @@ module.exports = class
         .then (credentials) ->
           options =
             encoding: opts.encoding
+            exportFormat: if opts.xlsx then 'xlsx' else 'csv'
```

Comment:
Hello @hisabimbola for JSON exports there is another tool - https://github.com/sphereio/sphere-node-cli